core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala

```diff
@@ -61,6 +61,14 @@ object RawlsApiService extends LazyLogging {
         )
         Sentry.captureException(rollback)
         complete(StatusCodes.InternalServerError -> ErrorReport(rollback))
+      case sql: SQLException =>
+        val sentryId = Sentry.captureException(sql)
```

Comment:
Running locally or in a BEE, this returns all zeros. Presumably in a real environment it will return the value with an actual UUID.

Reply:
Will you be able to test this once it is merged to dev?

Reply:
🤦 
I was thinking I would, because it would have sentry hooked up properly.
But when testing in a BEE, I added specific code to throw the exception directly.

So now that I think about it, I don't know how, without intentionally breaking the database or something. I'm definetly open to ideas.

Reply:
I wonder if it would be possible to muck with a locally running Rawls (which uses  dev databases) to do something so that dev Rawls throws a SQLException.

Or maybe there is a way to just test what `Sentry.captureException()` returns in dev independently.

Reply:
I guess I could just change something in the DB to an invalid value. I'm not sure that's an `SQLException` though - it might be some sort of slick exception.
FWIW, I did verify that when sentry is _not_ hooked up, `Sentry.captureException` still returns a non-null `SentryId`, which converts to a valid string. It isn't useful, but it at least isn't going to break anything.

Reply:
It seems like it might be worth attempting to change a DB value, since it would be a quick experiment.

Reply:
You could try testing it locally with the correct DSN for dev and see if a real connection to Sentry yields a nonzero UUID. IIRC dev Rawls isn't actually hooked up to Sentry and may not yield useful information.